2-mandatory/1-writers.js

```diff
@@ -71,6 +72,11 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
+  writers.forEach(o => {
```

Comment:
In light of the CYF guidelines regarding JavaScript, 
we need to get into the habit of using longer, descriptive variable names.

Instead of 'o'

Hint: https://syllabus.codeyourfuture.io/guides/code-style-guide#think-of-good-names-for-your-variables